src/json-symtab-language/Makefile

```diff
@@ -0,0 +1,17 @@
+SRC = json_symtab_language.cpp \
+	json_symbol_table.cpp \
+	json_symbol.cpp 
+
+INCLUDES= -I ..
+
+include ../config.inc
+include ../common
+
+CLEANFILES = json_symtab_language$(LIBEXT)
```

Comment:
For consistency with other directories/libs and the directory name,  this should probably be `json-symlab-language$(LIBEXT)` - but note you'll need to change that in all the makefiles/targets where it's used, of course.

Reply:
And note that it's *just* the library name that needs to change, *not* any source file names.

Reply:
Yes, you're right. This bit me in more than a couple of occasions, because some Makefiles refer to it as `json_symtab_language` and some others as `json-symtab-language`. I will try to change it to the second in all of them.